tests/shared/hooks.js

```diff
@@ -1,12 +1,17 @@
-const {generateSwarm} = require('../../src/daemonManager');
+const {generateSwarm, swarmManager} = require('../../src/daemonManager');
+const swarmRegistry = require('../../src/swarmRegistryAdapter');
 
 exports.remoteSwarmHook = function ({createDB = true} = {}) {
     before('initialize client and setup db', async function () {
-        this.api = bluzelle({
+
+        this.api = await bluzelle({
```

Comment:
Could this (and similar calls) use `clientManager.initializeClient`?

Reply:
Yeah, there's definitely some clean up to be done between tests/shared/hooks.js and clientManager.initializeClient. Will address this.